pkg/arvo/app/chat-cli.hoon

```diff
@@ -0,0 +1,1111 @@
+::  chat-cli: cli chat client using chat-store and friends
+::
+::    pulls all known messages into a single stream.
+::    type ;help for usage instructions.
+::
+::    note that while the chat-store only cares about paths,
+::    we mostly deal with [ship path] (aka target) here.
+::    when sending messages (through the chat hook),
+::    we concat the ship onto the head of the path,
+::    and trust it to take care of the rest.
+::
+/-  sole-sur=sole, *chat-store, *chat-view, *chat-hook
+/+  sole-lib=sole
+/=  seed  /~  !>(.)
```

Comment:
What does this do?

Reply:
It, uh, packages the subject up for us to use [later on in the file](https://github.com/urbit/urbit/blob/12050d44a827c0b6bca36f4547536ce39d3b3ba7/pkg/arvo/app/chat-cli.hoon#L613-L618)? I've been cargo-culting this over since 2016, evaluating some hoon in its own temporary context is arcane enough for me to break it if I try to change too many things.

Reply:
@belisarius222 is right, this should just be `!>(..zuse)`. In fact, the whole `+eval ` gate below should probably be moved into a library so it can be shared (there's an inferior version in `lib/chat-json`). I also think it's worth including `[our eny now]` in the eval subject.

/cc @loganallenc

Reply:
With the addition of `lib/chat/eval`, this line is now unused. Please remove.